src/cudamatrix/cu-kernels-ansi.h

```diff
@@ -108,8 +108,10 @@ void cudaF_min_mat_cols(int Gr, int Bl, float* result, const float* mat, const M
 void cudaF_sum_mat_cols(int Gr, int Bl, float* result, const float* mat, const MatrixDim d);
 void cudaF_replace_value(int Gr, int Bl, float *v, int dim, float orig, float changed);
 void cudaF_set_bias_params(int Gr, int Bl, float* v, const float* a, float param_1, float param_2, float param_3, int* flag, int dim);
-void cudaF_copy_from_vec_df(int Gr, int Bl, double* v_out, const float* v_in, int dim);
-void cudaF_copy_from_vec_fd(int Gr, int Bl, float* v_out, const float* v_in, int dim);
+void cublas_copy_kaldi_fd(int Gr, int Bl, int n, const float* x, 
```

Comment:
Thanks for your comments. Have fixed this issue. I checked the trailing whitespace using "/\s\+$" and remove those spaces using ":%s/\s\+$//e" in each file. Is there any better way to check and remove them? 
BTW, I encountered some unresolved merge conflict issues by doing what you suggested (my local brunch is out of date). To make life easier, I created a new PR #913, which is same as this one. I have tested the modified codes. Sorry for this inconvenience. 
Ke